dip-0004.md

```diff
@@ -94,16 +94,30 @@ A SML entry consists of the following fields:
 | pubKeyOperator | BLSPubKey | 48 | The operators public key |
 | keyIDVoting | CKeyID | 20 | The public key hash used for voting. |
 | isValid | bool | 1 | True if a masternode is not PoSe-banned |
+
```

Comment:
Seems like we're splitting 2 sets of constraints with a table here. Do we really mean smth like:
"After v19 activates, entries where `nVersion` equals 2 will be serialised for clients with protocol versions greater than or equal to 70227."